src/setupTests.ts

```diff
@@ -1,4 +1,2 @@
 import '@testing-library/jest-dom';
 import './tests/initI18n';
-
-process.env.SASS_PATH = './src/assets/styles';
```

Comment:
Is there a reason to move SASS_PATH definition back to package.json?

Reply:
Yeah, that one wasn't actually working, it was some cache that made it look like it was working the other day :(. Spend lot of time to yesterday trying to get that env variable working some other way than in package.json but couldn't.

Reply:
Ah, ok. In that case, it's better to go with working solution